<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableWidget" name="tw_view">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>761</width>
        <height>311</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>350</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>투표목록</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>투표수</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="le_input">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbt_input">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>등록</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbt_start">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> 투표하기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbt_view">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>기록보기</string>
      </property>
     </widget>
     <widget class="plotWidget" name="graph1" native="true">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>581</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbt_erase">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>지우기</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>투표 회차</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tw2_view">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>761</width>
        <height>311</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>350</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>투표목록</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>투표수</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pbt_input_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>home</string>
      </property>
     </widget>
     <widget class="QWidget" name="graph2" native="true">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>581</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>plotWidget</class>
   <extends>QWidget</extends>
   <header location="global">plotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
